lib/graticule/geocoder/mapbox.rb

```diff
@@ -1,7 +1,8 @@
-module Graticule
-  module Geocoder
+require 'json'
```

Comment:
What changed that we need to `require 'json'`?

Reply:
Oh good catch. Nothing… was trying to be explicit when trying to track down some issues during testing.